--- giblib-hak/configure.ac	2023-03-17 19:52:43.520815693 -0400
+++ giblib-1.2.4/configure.ac	2004-09-04 09:42:10.000000000 -0500
@@ -17,6 +17,10 @@
 AM_MAINTAINER_MODE
 AM_WITH_DMALLOC
 
-AC_PATH_GENERIC(imlib2, 1.0.0, [
-  AC_SUBST(IMLIB_LIBS)
-  AC_SUBST(IMLIB_CFLAGS) ],
-  AC_MSG_ERROR(Cannot find imlib2: Is imlib2-config in the path?) )
 
 dnl the above doesn't work for some reason :/
 IMLIB_LIBS=`imlib2-config --libs`
